k8s/overlays/coreso/test/env.properties

```diff
@@ -4,62 +4,71 @@ GRIDCAPA_SOURCE_HOST_CSE_IMPORT_D2CC=192.168.236.230
 GRIDCAPA_SOURCE_PORT_CSE_IMPORT_D2CC=21
 GRIDCAPA_SOURCE_BASE_DIRECTORY_CSE_IMPORT_D2CC=/cse/import/d2cc
 GRIDCAPA_SINK_DIRECTORY_CSE_IMPORT_D2CC=/cse/import/d2cc/outputs
+GRIDCAPA_POLLING_DELAY_CSE_IMPORT_D2CC=180000
```

Comment:
It's better to add these parameters also for Coreso prod for the next release

Reply:
Do you mean only add the new values in the env.properties file for prod, or also add environment variable in deployments?

Reply:
Both of course ! You can keep the current value 60 s for all processes like it's in the current version installed